recipes/xarray-einstats/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Generated using grayskull; edited by @ericmjl.
+{% set name = "xarray-einstats" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xarray-einstats-{{ version }}.tar.gz
+  sha256: 5c4e00d09172daf665cbd5e533a897896b84f37feb5cab0fd75829477fbcda7d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
```

Comment:
as the library has pyproject.toml only and no setup.py nor setup.cfg, it needs pip>=21.3, it also looks like flit should be installed for everything to work as it is a build system requirement (which I think means it should go here in the host)

Reply:
Great catch, thank you, @OriolAbril!